clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverExecutor.scala

```diff
@@ -46,9 +46,10 @@ class DeliverExecutor[CI <: DeliverableIndex](
   }
 
   override protected[dispatch] def buildMove(
-    metadata: moveCommand.index.MetadataType
+    metadata: moveCommand.index.MetadataType,
+    undeliver: Boolean = false
   ): Source[(moveCommand.index.MetadataType, immutable.Seq[IoOp]), NotUsed] = {
-    super.buildMove(metadata).map {
+    super.buildMove(metadata, undeliver).map {
```

Comment:
If this works, then you should be able to back this change out

Reply:
Yup, you're completely right. Clio is confusing when you're deep in it...